### ISSUE TITLE
Issues with VS Code Project Commands page

### ISSUE THREAD

**Issue by tetchel** [eclipse/codewind#632]
I was looking through https://www.eclipse.org/codewind/mdt-vsc-commands-project.html when adding a link to the Project Settings sections and noticed a few issues.

1. "Open folder as workspace:"
This command has been removed, and replaced with the more intuitive "Add Project to Workspace". Here's a description for the new command:
```
Add Project to Workspace: Add this project's folder on your local disk to your VS Code workspace.
```

2. Open Performance Dashboard
This command is missing! We have it in Eclipse, though [here](https://www.eclipse.org/codewind/mdteclipsemanagingprojects.html):
```
Opens the performance dashboard in the default Eclipse browser. This action is available only when the application is running or debugging.
```

We can copy this description into VS Code, except removing the words "default Eclipse". Add it under the Open Application Monitor action.

3. We should add the Remove Project command as follows:
```
Remove Project: Removes the project from Codewind. The project folder can optionally be deleted from disk after removal.
```

Add it at the bottom of the Commands list.

4. There's a weirdly large gap of whitespace between the Commands and Logs sections.

5. Under **Logs**:
```
Project-specific settings: Edit the internal app and debug ports as well as the application endpoint, sometimes known as the context root, from the Project Overview page.
```

This looks like a straight mistake, there's a section about project-specific settings below. This definitely does not belong in the logs section; please delete it.

**Comment by micgibso** [comment]
@sishida I'll take a look at this one, thanks

**Comment by micgibso** [comment]
@tetchel See PR https://github.com/eclipse/codewind-docs/pull/172

**Comment by tetchel** [comment]
fixed in eclipse/codewind-docs#172
